generated/nixnet/nixnet.proto

```diff
@@ -0,0 +1,1239 @@
+  DB_PROPERTIES_CLST_APPLICATION_PROTOCOL = 65651;
+  DB_PROPERTIES_CLST_BAUD_RATE = 65537;
+  DB_PROPERTIES_CLST_CAN_FD_BAUD_RATE_64 = 151060497;
+  DB_PROPERTIES_CLST_CAN_FD_BAUD_RATE = 65553;
+  DB_PROPERTIES_CLST_CAN_FD_ISO_MODE = 65652;
+  DB_PROPERTIES_CLST_CAN_IO_MODE = 65552;
+  DB_PROPERTIES_CLST_COMMENT = 50397192;
+  DB_PROPERTIES_CLST_CONFIG_STATUS = 65545;
+  DB_PROPERTIES_CLST_DATABASE_REF = 83951618;
+  DB_PROPERTIES_CLST_ECU_REFS = 100728835;
+  DB_PROPERTIES_CLST_FLEX_RAY_ACT_PT_OFF = 65568;
+  DB_PROPERTIES_CLST_FLEX_RAY_ALW_PASS_ACT = 65650;
+  DB_PROPERTIES_CLST_FLEX_RAY_CAS_RX_L_MAX = 65569;
+  DB_PROPERTIES_CLST_FLEX_RAY_CHANNELS = 65570;
+  DB_PROPERTIES_CLST_FLEX_RAY_CLST_DRIFT_DMP = 65571;
+  DB_PROPERTIES_CLST_FLEX_RAY_COLD_ST_ATS = 65572;
+  DB_PROPERTIES_CLST_FLEX_RAY_CYCLE = 65573;
+  DB_PROPERTIES_CLST_FLEX_RAY_DYN_SEG_START = 65574;
+  DB_PROPERTIES_CLST_FLEX_RAY_DYN_SLOT_IDL_PH = 65575;
+  DB_PROPERTIES_CLST_FLEX_RAY_LATEST_GUAR_DYN = 65579;
+  DB_PROPERTIES_CLST_FLEX_RAY_LATEST_USABLE_DYN = 65578;
+  DB_PROPERTIES_CLST_FLEX_RAY_LIS_NOISE = 65576;
+  DB_PROPERTIES_CLST_FLEX_RAY_MACRO_PER_CYCLE = 65577;
+  DB_PROPERTIES_CLST_FLEX_RAY_MACROTICK = 16842800;
+  DB_PROPERTIES_CLST_FLEX_RAY_MAX_WO_CLK_COR_FAT = 65585;
+  DB_PROPERTIES_CLST_FLEX_RAY_MAX_WO_CLK_COR_PAS = 65586;
+  DB_PROPERTIES_CLST_FLEX_RAY_MINISLOT = 65588;
+  DB_PROPERTIES_CLST_FLEX_RAY_MINISLOT_ACT_PT = 65587;
+  DB_PROPERTIES_CLST_FLEX_RAY_NIT = 65590;
+  DB_PROPERTIES_CLST_FLEX_RAY_NIT_START = 65591;
+  DB_PROPERTIES_CLST_FLEX_RAY_NM_VEC_LEN = 65589;
+  DB_PROPERTIES_CLST_FLEX_RAY_NUM_MINISLT = 65592;
+  DB_PROPERTIES_CLST_FLEX_RAY_NUM_STAT_SLT = 65593;
+  DB_PROPERTIES_CLST_FLEX_RAY_OFF_COR_ST = 65600;
+  DB_PROPERTIES_CLST_FLEX_RAY_PAYLD_LEN_DYN_MAX = 65601;
+  DB_PROPERTIES_CLST_FLEX_RAY_PAYLD_LEN_MAX = 65602;
+  DB_PROPERTIES_CLST_FLEX_RAY_PAYLD_LEN_ST = 65603;
+  DB_PROPERTIES_CLST_FLEX_RAY_STAT_SLOT = 65605;
+  DB_PROPERTIES_CLST_FLEX_RAY_SYM_WIN = 65606;
+  DB_PROPERTIES_CLST_FLEX_RAY_SYM_WIN_START = 65607;
+  DB_PROPERTIES_CLST_FLEX_RAY_SYNC_NODE_MAX = 65608;
+  DB_PROPERTIES_CLST_FLEX_RAY_TSS_TX = 65609;
+  DB_PROPERTIES_CLST_FLEX_RAY_USE_WAKEUP = 33620053;
+  DB_PROPERTIES_CLST_FLEX_RAY_WAKE_SYM_RX_IDL = 65616;
+  DB_PROPERTIES_CLST_FLEX_RAY_WAKE_SYM_RX_LOW = 65617;
+  DB_PROPERTIES_CLST_FLEX_RAY_WAKE_SYM_RX_WIN = 65618;
+  DB_PROPERTIES_CLST_FLEX_RAY_WAKE_SYM_TX_IDL = 65619;
+  DB_PROPERTIES_CLST_FLEX_RAY_WAKE_SYM_TX_LOW = 65620;
+  DB_PROPERTIES_CLST_FRM_REFS = 100728836;
+  DB_PROPERTIES_CLST_LIN_SCHEDULES = 100728944;
+  DB_PROPERTIES_CLST_LIN_TICK = 16842865;
+  DB_PROPERTIES_CLST_NAME = 50397189;
+  DB_PROPERTIES_CLST_PDU_REFS = 100728840;
+  DB_PROPERTIES_CLST_PD_US_REQD = 33619978;
+  DB_PROPERTIES_CLST_PROTOCOL = 65542;
+  DB_PROPERTIES_CLST_SIG_REFS = 100728839;
+  DB_PROPERTIES_FRM_APPLICATION_PROTOCOL = 131172;
+  DB_PROPERTIES_FRM_CAN_EXT_ID = 33685520;
+  DB_PROPERTIES_FRM_CA_NIO_MODE = 131174;
```

Comment:
I think these are `CAN_IO`. The input was `CANio`, which is a problem for the name converter. So keep an eye out for any  names like that. It looks that's xnet's way to deal with back-to-back acronyms, at least in this case. If it looks like there are a bunch of them, we can consider automating the detection of that (@DavidCurtiss wrote it for RFmx). The volume is way lower for xnet. And the look relatively well behaved. Although the abbreviated names are pretty cryptic.